backend/azureblob/azureblob.go

```diff
@@ -217,6 +218,17 @@ This option controls how often unused buffers will be removed from the pool.`,
 				encoder.EncodeDel |
 				encoder.EncodeBackSlash |
 				encoder.EncodeRightPeriod),
+		}, {
+			Name: "public_access",
+			Help: `Public access level of a container: blob, container or private(default).
+
+This sets the public access level of a container. Default is none that means only authorized user can get access to the container.
+Other options are "blob" and "container" where
+container: Specifies full public read access for container and blob data. Clients can enumerate blobs within the container via anonymous request,
```

Comment:
The docs for `container` etc might be better done as an examples list - this will let the user choose in the configurator and they will be shown in the docs, eg

https://github.com/rclone/rclone/blob/cc08f66dc178776b44faabdb90d1846a0c525fa7/backend/crypt/crypt.go#L39-L49